opencti-platform/opencti-front/src/private/components/data/ingestionCsv/IngestionCsvLine.tsx

```diff
@@ -57,6 +57,7 @@ const ingestionCsvLineFragment = graphql`
     uri
     ingestion_running
     current_state_date
```

Comment:
I think this date is never valued. Am I missing something ?

Reply:
It's working if you put a date in the "import from date" at the creation, but I just noticed that the date update doesn't work